.github/workflows/wit-wasmcloud-postgres.yml

```diff
@@ -12,6 +12,9 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
I think you _might_ need
```
contents: write
```
here, since the goal is to create a release not a PR right?

Reply:
Ah sorry yes -- I need contents write *only*! `pull-requests` is entirely the wrong one